### PR TITLE
Use publish-app-id in pubsub quickstart.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Use publish-app-id in pubsub quckstart.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/cli/issues/536

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [X] You've updated the quickstart's README if necessary

### CODE REVIEW

File: pub-sub/README.md [hunk 1]
```diff
@@ -98,19 +98,19 @@ The Dapr CLI provides a mechanism to publish messages for testing purposes. Now,
   * Linux/Mac:
 
 ```bash
-dapr publish --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
+dapr publish --publish-app-id python-subscriber --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
```
Comment (mukundansundar) [inline pub-sub/README.md:101]: `publish-app-id` is supposed to be used with the publisher app and not the subscriber app. Can we run the react form app which is the publisher in this case, show the CLI way of publishing events and the show the GUI?

Comment (artursouza) [reply pub-sub/README.md:101]: /cc @msfussell

Comment (msfussell) [reply pub-sub/README.md:101]: LGTM

Comment (artursouza) [reply pub-sub/README.md]: Please, see how it reads now with the latest version.

**Review by msfussell — DISMISSED** [review]
This is a better flow, now the command needs publish command needs the --app-id name.

File: pub-sub/README.md [hunk 2]
```diff
@@ -89,63 +89,63 @@ python -m pip install -r requirements.txt
...
+1. Use Dapr CLI to publish a message:
+   
+  * Linux/Mac:
 
 ```bash
-cd react-form
-```
 
-2. Run the React front end app with Dapr: 
+dapr publish --topic A --pubsub react-form --data '{ "message": "This is a test" }'
```
Comment (mukundansundar) [inline pub-sub/README.md:130]: i think pubsub should be the same `pubsub` and the publish-app-id should be `react-form`

**Review by mukundansundar — CHANGES_REQUESTED** [review]
i think pubsub should be the same `pubsub` and the publish-app-id should be `react-form`

File: pub-sub/README.md (hunk 2 above)
Comment (artursouza) [reply pub-sub/README.md:130]: My mistake.

Comment (artursouza) [reply pub-sub/README.md]: Fixed.

**Review by mukundansundar — APPROVED** [review]
lgtm

**Pull request merged**